rfcs/20220221-3d_model_support.md

```diff
@@ -0,0 +1,211 @@
+# RFC: 3D Model Support
+
+- [ ] [Approver TBD]
```

Comment:
Tentatively approving. The backend plan seems good to me. I think we'll want another RFC with more specific implementation details once we have frontend designs but this has been a really good starting point for having us discuss and think about the various hurdles we'll need to make sure designs are aware of.

```suggestion
- [z] @sarayourfriend
```